package-system/vulkan-validationlayers/Findvulkan-validationlayers.cmake.stub

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+set(MY_NAME "vulkan-validationlayers")
+set(TARGET_WITH_NAMESPACE "3rdParty::${MY_NAME}")
+if (TARGET ${TARGET_WITH_NAMESPACE})
+    return()
+endif()
+# Blank stub library for Linux and Android as they are not supported yet.
+add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
```

Comment:
The stub seems like a bad way to do it, irght? it either means having to rev the windows package later when you support these, or, having to ship empty stub packages to people unnecessraily.

Instead of doing a stub, you could put the dependency and declaration of ly_associate-package on 3rdParty::vulkan-validationlayers in O3DE source in a PAL file where its needed and only on the platforms its needed on.  You could even define something in there if you needed code to know whether its there or not.